index.html

```diff
@@ -18,5 +18,69 @@
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-  </body>
+  
+      <header class="header">
+
+        <a href="#">
+          <img
+            src="css/arsenal logo 2.png"
+            alt="Logo Arsenal"
+            class="header__logo"
+          />
+        </a>
+
+       <nav>
+          <ul class="navigation__list">
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">Home Games</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">Away Games</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">Store</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">Contact Us</a>
+            </li>
+          </ul>
+        </nav>
+
+      
+      </header>
+      <main>
+        <section class="fans">
+          <h1 class="fans">Arsenal Home Fans Page</h1>
+          <h3 class="history">
```

Comment:
Try not to skip `h` levels within a section as it throws off the structure of the page (e.g the next `h` from `h1` should always be a `h2`)